draft-ietf-quic-transport.md

```diff
@@ -2345,7 +2345,7 @@ following layout:
  0                   1                   2                   3
  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|0|1|               Unpredictable Bits (182..)                ...
+|0|1|               Unpredictable Bits (422..)                ...
```

Comment:
So we're increasing the min size of a stateless reset token?  I'd really rather not, since that increases the chances of that being a DoS vector.  Keeping the ratio of incoming to outgoing bytes ~1:1 seems important for DoS.  As a server operator, I don't want to not respond to a client's PING only packet with a stateless reset just because it's too small, but I also don't want to respond to every small packet I don't have state for.

Reply:
I still think we should allow responding to smaller packets at some random factor rather than not responding at all. This does not mean we should not try to keep resets small though.

Reply:
#2770 filed as a separate issue for this.

Reply:
OK, if you agree that this issue is non-blocking I'll proceed.